exercises/bob/.meta/generator/bob_case.rb

```diff
@@ -0,0 +1,9 @@
+require 'generator/exercise_case'
+
+class BobCase < Generator::ExerciseCase
+  def workload
+    indent_lines(["remark = %Q(#{input})",
+      "assert_equal '#{expected}', Bob.hey(remark), %Q(Bob hears #{input}, and..)"
```

Comment:
Try doing the same with the `%Q` but instead use `%\Q` to escape it.  That should reconstruct the executable line, but ignore it in place.

Let me know.